app/maui-blazor/MauiProgram.cs

```diff
@@ -0,0 +1,47 @@
+﻿using CommunityToolkit.Maui;
+using CommunityToolkit.Maui.Media;
+
+namespace MauiBlazor;
+
+public static class MauiProgram
+{
+	public static MauiApp CreateMauiApp()
+	{
+		var builder = MauiApp.CreateBuilder();
+		builder
+			.UseMauiApp<App>()
+			.ConfigureFonts(fonts =>
+			{
+				fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+			});
+
+		builder.UseMauiCommunityToolkit();
+
+		builder.Services.AddMauiBlazorWebView();
+
+#if DEBUG
+		builder.Services.AddBlazorWebViewDeveloperTools();
+		builder.Logging.AddDebug();
+#endif
+
+		builder.Services.AddHttpClient<ApiClient>(client =>
+		{
+			// TODO: Configure this to point to your deployed API. For example, https://MY_HOSTED_APP.example.azurecontainerapps.io/
+			client.BaseAddress = new Uri("TODO");
```

Comment:
Is this `TODO` going to be addressed before we approve and merge? Is this something that can be done easily, how is that usually handled with .NET MAUI apps?

Reply:
For now this is something that the person running the app needs to do. I think it's mentioned in the README comments. This URL needs to be set to wherever the app is published on Azure. There's no automatic way to do that _yet_ but we're looking at possible product features that would enable this automatically in the future (perhaps you saw some earlier thinkings on this by having something like `azd up` set it automatically).

Reply:
Added a better comment